src/main/java/net/openhft/chronicle/wire/TextWire.java

```diff
@@ -1932,7 +1932,7 @@ public <T> WireIn uint16(@NotNull T t, @NotNull ObjIntConsumer<T> ti) {
         @Override
         public <T> WireIn int32(@NotNull T t, @NotNull ObjIntConsumer<T> ti) {
             consumePadding();
-            ti.accept(t, (int) getALong());
+            ti.accept(t, (int) int64());
```

Comment:
Why does this accept int64 when the calling method is called int32?

(I see this is not a new issue, it used to be a long before)

Reply:
I really have no answer but I can add "enforce or throw" check here. I'm not advising to backport this change anyway and it seems safe to do in `ea`. WDYT?